### PR TITLE
adds laser arm implant to robotics only uplink

### PR DESCRIPTION
**Pull request by improvedname**
will the mad traitor make robocop
or will he BE robocop.

tottaly balanced could do better with a ebow and a esword anyways
detectable by every medhuds instantly

:cl: Improvedname
add: Traitor roboticists can now buy laser arms
/:cl:

### CODE REVIEW

**Comment by Wilchenx** [comment]
You could buy one useless arm for 20tc

Or you could buy emag for 8 and have a army of borgs.

It's just a pointless addition with its current functionality and cost.

**Comment by improvedname** [comment]
you could murderbone or you could be a cool hightech robocop traitor

**Comment by improvedname** [comment]
its the ARPEE experience

**Comment by improvedname** [comment]
besides >needing a emagg to get a army of cyborgs

**Comment by Grazyn** [comment]
The price seems a bit too high for a not-so-hidden lasergun that can be disabled by anyone hitting you once with a screwdriver.  Unless it's meant to be just a meme like the syndie balloon, but in that case it feels like a wasted job-specific item slot. If you want to balance it around emagged borgs, you could just make so that you can't buy both

**Comment by improvedname** [comment]
@Grazyn the lasergun litteraly retracts into your arm like a toolarm its as hidden as it can be and screwdrivering doesnt disable it

**Comment by Ratvar** [comment]
Make implant be usable on both hands at once? Having two pew-pews sounds much cooler, and, due to having bad accuracy and even more EMP weakness, isn't as unbalanced

**Comment by AlexTheSysop42** [comment]
if you're making it 20tc at least make it the full combat arm implant that has laser, flash, medgun, and eblade

also @Grazyn the thing that can be disabled with a screwdriver is just the autoimplanter, it's also used for nukeop implants

File: code/modules/surgery/organs/autosurgeon.dm [hunk 1]
```diff
@@ -79,6 +79,11 @@
 	desc = "A single use autosurgeon that contains a medical heads-up display augment. A screwdriver can be used to remove it, but implants can't be placed back in."
 	uses = 1
 	starting_organ = /obj/item/organ/cyberimp/eyes/hud/medical
+	
+/obj/item/autosurgeon/robotics
+	desc = "A single use autosurgeon that contains a combat arm implant augment. A screwdriver can be used to removte it, but implants can't be placed back in."
```
Comment (TBartl) [inline code/modules/surgery/organs/autosurgeon.dm]: "removte"

**Comment by SpaceManiac** [comment]
>can be disabled by anyone hitting you once with a screwdriver

"Use a screwdriver to remove it." refers to removing the implant from the autosurgeon, not from the person.

The uplink description needs a period at the end.

Otherwise looks good to me.

**Comment by Tlaltecuhtli** [comment]
bloat for sake of bloat

**Comment by improvedname** [comment]
Ill get on dis laterr...

**Comment by GunHog** [comment]
Well, it will let them borg people _faster!_

**Comment by davethwave** [comment]
AND it can allow for some fun things with the poison kit. paralyze people and give them nanomachines, or something similar.

**Comment by PKPenguin321** [comment]
This is actually a pretty good way to implement these

**Pull request closed without merging**